draft-ietf-privacypass-auth-scheme.md

```diff
@@ -587,33 +587,39 @@ generated using a cryptographically secure source of randomness ({{?RFC4086}}).
 
 ## Replay Attacks
 
+Applications SHOULD constrain tokens to a single origin unless the use case can
+accommodate such replay attacks. Replaying tokens is not necessarily a security
```

Comment:
```suggestion
accommodate replay attacks. Replaying tokens is not necessarily a security
```

Since we don't list the attacks before, they're not "such"